code/modules/projectiles/guns/ego_gun.dm

```diff
@@ -28,6 +33,25 @@
 		. += "<span class='notice'>[special]</span>"
 	if(weapon_weight != WEAPON_HEAVY)
 		. += "<span class='notice'>This weapon can be fired with one hand.</span>"
+
+	if(reloadtime)
+		. += "Ammo Counter: [shotsleft]/[initial(shotsleft)]."
+	else
+		. += "This weapon has unlimited ammo."
+
+	if(reloadtime)
+		switch(reloadtime)
+			if(1 to 0.7 SECONDS)
```

Comment:
Uh, I assume this was a typo or something since I doubt you intended this to have an interval of 1 second considering how the next if statement has 1 second in the interval.

Reply:
Right, should be a 0. one moment.